rust/src/transformations/quantile_score_candidates/make_quantile_score_candidates.tex

```diff
@@ -0,0 +1,318 @@
+        \\&\hspace{1cm}|\min(\#(X' < C_i), l) - \alpha \cdot \min(|X'| - \#(X' = C_i), l)|| \\
+    &\le \alpha_{den} \cdot \max\limits_{i} ||\#(X < C_i) - \alpha \cdot (|X| - \#(X = C_i))|
+        \\&\hspace{1cm}|\#(X' < C_i) - \alpha \cdot (|X| - \#(X' = C_i))|| \\
+\intertext{Consider each of the three cases of adding or removing an element in X.}
+\intertext{Case 1. Assume $X'$ is equal to $X$, but with some $X_j < C_i$ added or removed.}
+    &= \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - |(1 - \alpha) \cdot (\#(X < C_i) \pm 1) - \alpha \cdot \#(X > C_i)|| \\
+    &\leq \alpha_{den} \cdot  \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)|
+        \\&\hspace{1cm} - (|(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| + |\pm(1 - \alpha)|)| &&\text{by triangle inequality} \\
+    &= \alpha_{den} \cdot \max\limits_{i} |1 - \alpha| &&\text{scores cancel}\\
+    &= \alpha_{den} - \alpha_{num} &&\text{since } \alpha \leq 1
+\intertext{Case 2. Assume $X'$ is equal to $X$, but with some $X_j > C_i$ added or removed.}
+    &= \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - |(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot (\#(X > C_i) \pm 1)|| \\
+    &\leq \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - (|(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| + |\pm\alpha|)| &&\text{by triangle inequality} \\
+    &= \alpha_{den} \cdot \max\limits_{i} |\alpha| &&\text{scores cancel}\\
+    &= \alpha_{num} &&\text{since } \alpha \geq 0
+\intertext{Case 3. Assume $X'$ is equal to $X$, but with some $X_j = C_i$ added or removed.}
+    &= \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - |(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)|| \\
+    &= 0 &&\text{no change in score}
+\intertext{Take the union bound over all cases.}
+    &\le \max(\alpha_{num}, \alpha_{den} - \alpha_{num})
+\end{align*}
+
+\end{proof}
+
+Take any two elements $X, X'$ in the \texttt{input\_domain} and any pair $(\din, \dout)$, 
+where \din\ has the associated type for \texttt{input\_metric} and \dout\ has the associated type for \texttt{output\_metric}.
+Assume $X, X'$ are \din-close under \texttt{input\_metric} and that $\texttt{stability\_map}(\din) \leq \dout$. 
+
+\begin{align*}
+    \dout &= \max\limits_{X \sim X'} d_{IDD}(s, s') &&\text{where } s = \function(X)\\
+    &= \max\limits_{X \sim X'} \max\limits_{ij} |(s_i - s'_i) - (s_j - s'_j)| &&\text{by definition of }\rustdoc{metrics/struct}{LInfDiffDistance} \\
+    &\leq 2 \max\limits_{X \sim X'} \max\limits_{i} |s_i - s'_i| &&\text{$s$ is not monotonic; take looser bound}\\
```

Comment:
I do not understand how to get to these 2 lines (?)

Reply:
Will update to make this clearer. The gist is that it is always true, by the triangle inequality, that 
```
|(s_i - s'_i) - (s_j - s'_j)| <= |(s_i - s'_i)| + |- (s_j - s'_j)| <= |(s_i - s'_i)| + |- (s_i - s'_i)| <= 2 \cdot |s_i - s'_i|
```